native/packages/react-native-bpk-component-navigation-bar/src/TitleView.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import BpkText from 'react-native-bpk-component-text';
+import BpkIcon, { icons } from 'react-native-bpk-component-icon';
+import { spacingSm, spacingLg } from 'bpk-tokens/tokens/base.react.native';
+
+import { type TitleWithIcon } from './common-types';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  leadingIcon: {
+    marginEnd: spacingSm,
+  },
+  trailingIcon: {
+    marginStart: spacingSm,
+  },
+  textWithIcon: {
+    // This line height ensure the icon lines up
+    // nicely with the baseline of the text.
+    lineHeight: spacingLg,
```

Comment:
I reckon this should be a "magic" number as opposed to a token. 

The use of a token makes it seems like the line height value has meaning. It actually doesn't - the value just so happened to work with lock icons. A different value is required for other icons AFIK. 

Does that make sense?